scripts/Password_Manager/main.py

```diff
@@ -0,0 +1,106 @@
+import random
+import base64
+import pickle
+from Crypto.Cipher import AES
+import hashlib
+
+
+def AutoGen(number):
+    passwd = ""
+    for i in range(number):
+        passwd += chr(random.randint(33, 126))
+    return passwd
+
+
+def encrypt_file(message):
+
+    password = "password".encode()
+
+    key = hashlib.sha256(password).digest()
```

Comment:
Same as before instead of hard coding the password, use a master password for the key used during encryption and decryption. 

```py
from PBKDF2 import PBKDF2
from Crypto.Cipher import AES
import os
salt = os.urandom(8)    # 64-bit salt
key = PBKDF2("This passphrase is a secret.", salt).read(32) # 256-bit key
iv = os.urandom(16)     # 128-bit IV
cipher = AES.new(key, AES.MODE_CBC, iv)
```
master password -> secure key-> encrypt data by the key

Reply:
okey, let me work on it now
I need your help too

Reply:
Sure thing.